config/constants.go

```diff
@@ -16,11 +16,33 @@ const (
 // Because Go does not support constant Structs :(
 func defaultConfig() Config {
 	return Config{
-		HistoryFilePath: defaultHistoryFile,
-		MaxHistory:      defaultMaxHist,
-		AllowDuplicates: defaultAllowDuplicates,
-		TempDirPath:     defaultTempDir,
-		LogFilePath:     defaultLogFile,
-		ThemeFilePath:   defaultThemeFile,
+		AllowDuplicates: true,
+		HistoryFilePath: "clipboard_history.json",
+		MaxHistory:      1000,
```

Comment:
I think default limit should be kept to 100, as having a large history can cause performance issues for some machines.